src/subscriber.ts

```diff
@@ -1,32 +1,39 @@
 import { ApiPromise, WsProvider } from '@polkadot/api';
-import { SessionIndex } from '@polkadot/types/interfaces';
+import { SessionIndex, Registration, IdentityInfo } from '@polkadot/types/interfaces';
 import { Logger } from '@w3f/logger';
 import { Text } from '@polkadot/types/primitive';
-
 import {
-    InputConfig
+    InputConfig, JudgementRequest, StorageData
 } from './types';
+import Event from '@polkadot/types/generic/Event';
+import { Option } from '@polkadot/types'
+import fs from 'fs'
+import storage from 'node-persist';
 
 export class Subscriber {
     private chain: Text;
     private api: ApiPromise;
     private endpoint: string;
     private sessionIndex: SessionIndex;
     private logLevel: string;
+    private readonly PERSISTENCE_DIR = './persistence'
 
     constructor(
         cfg: InputConfig,
         private readonly logger: Logger) {
         this.endpoint = cfg.endpoint;
+        this.logLevel = cfg.logLevel;
     }
 
     public async start(): Promise<void> {
         await this._initAPI();
+        await this._initPersistence();
         await this._initInstanceVariables();
 
-        if(this.logLevel === 'debug') await this._triggerDebugActions()
+        if(this.logLevel == 'debug') await this._triggerDebugActions()
```

Comment:
Do we really need this check? If the log level is higher than debug, the debug actions messages won't be displayed.

Reply:
Correct, but I'm using the logLevel even to trigger _triggerDebugActions, a set of actions useful to the development phase ( it is not just a matter of log messages ). Doing so just for convenience, I could use a dedicated flag such as development: true from the config. Wdyt ?

Reply:
It's ok like this, don't spend too much time on it, we can refactor later :+1: